### PR TITLE
Add new proc for Content View Version Comparison

### PR DESCRIPTION
**Pull request by bangelic**
New procedure added for content view version comparison.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8, orcharhino 6.1 on EL7, orcharhino 6.2 on EL7/8)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/proc_content-view-version-comparison.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[id="Content_View_Version_Comparison_{context}"]
+= Content View Version Comparison
+
+Use this procedure to compare Content View Version functionality for {Project}.
+
+.Procedure
+* In the {Project} web UI, navigate to *Content* > *Content Views* >  then view list of versions to select two versions and click Compare.
+
```
Comment (adamlazik1) [inline guides/common/modules/proc_content-view-version-comparison.adoc:8]:
```suggestion
* In the {ProjectWebUI} , navigate to *Content* > *Content Views*, then select two Content View versions and click Compare.
```

File: guides/common/modules/proc_content-view-version-comparison.adoc [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+The compare screen will have pre-selected versions in the version dropdown menus and tabs for all content types found in either version.
```
Comment (adamlazik1) [inline guides/common/modules/proc_content-view-version-comparison.adoc:9]:
```suggestion
The compare screen has the pre-selected versions in the version dropdown menus and tabs for all content types found in either version.
```

File: guides/common/modules/proc_content-view-version-comparison.adoc [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+You can filter the results to show only the same, different, or all content types.
+You can select different versions to compare on this screen by selecting them from the version dropdowns.
```
Comment (adamlazik1) [inline guides/common/modules/proc_content-view-version-comparison.adoc:11]:
```suggestion
You can compare different Content View versions by selecting them from the dropdown menus.
```

File: guides/common/modules/proc_content-view-version-comparison.adoc [hunk 4]
```diff
@@ -0,0 +1,11 @@
+[id="Content_View_Version_Comparison_{context}"]
+= Content View Version Comparison
```
Comment (adamlazik1) [inline guides/common/modules/proc_content-view-version-comparison.adoc:2]:
```suggestion
[id="Comparing_Content_View_Versions_{context}"]
= Comparing Content View Versions
```
Editing title to gerund form by the IBM style guide. Please also update file name

File: guides/common/assembly_managing-content-views.adoc [hunk 5]
```diff
@@ -25,3 +25,5 @@ include::modules/proc_creating-a-content-filter-for-apt-content.adoc[leveloffset
 endif::[]
 
 include::modules/proc_deleting-multiple-content-view-versions.adoc[leveloffset=+1]
+
+include::modules/proc_content-view-version-comparison.adoc[leveloffset=+1]
```
Comment (adamlazik1) [inline guides/common/assembly_managing-content-views.adoc:29]:
```suggestion
include::modules/proc_comparing-content-view-versions.adoc[leveloffset=+1]
```

**Review by adamlazik1 — COMMENTED** [review]
Added a few suggestions on styling.

File: guides/common/modules/proc_comparing-content-view-versions.adoc [hunk 6]
```diff
@@ -0,0 +1,11 @@
+[id="Comparing_Content_View_Versions_{context}"]
+= Comparing Content View Versions
+
+Use this procedure to compare Content View Version functionality for {Project}.
+
+.Procedure
+* In the {ProjectWebUI}, navigate to *Content* > *Content Views*, then select two Content View versions and click Compare.
+
+The compare screen has the pre-selected versions in the version dropdown menus and tabs for all content types found in either version.
```
Comment (adamlazik1) [inline guides/common/modules/proc_comparing-content-view-versions.adoc:9]: LGTM! One last suggestion to align the paragraph, and then I believe this is good to go.

```suggestion
+
The compare screen has the pre-selected versions in the version dropdown menus and tabs for all content types found in either version.
```

**Review by vsedmik — APPROVED** [review]
Looks good to me, ACK

**Comment by bangelic** [comment]
@maximiliankolb Would you be able to merge this PR? It's been ACK from QE.

File: guides/common/modules/proc_comparing-content-view-versions.adoc [hunk 7]
```diff
@@ -0,0 +1,11 @@
+[id="Comparing_Content_View_Versions_{context}"]
+= Comparing Content View Versions
+
+Use this procedure to compare Content View Version functionality for {Project}.
+
+.Procedure
+* In the {ProjectWebUI}, navigate to *Content* > *Content Views*, then select two Content View versions and click Compare.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_comparing-content-view-versions.adoc]:
```suggestion
* In the {ProjectWebUI}, navigate to *Content* > *Content Views*, then select two Content View versions and click *Compare*.
```

**Comment by maximiliankolb** [comment]
@bangelic one small suggestion, then we're good to go!

**Comment by bangelic** [comment]
@maximiliankolb Done! Thank you for your suggestion!

**Comment by adamlazik1** [comment]
Merged and cherry-picked:
   960345af3..63dc374a5  3.5 -> 3.5

**Pull request merged**